docs/topics/tour/kotlin-tour-control-flow.md

```diff
@@ -186,6 +189,85 @@ You can also do the same with `Char` ranges:
 * `'a'..'d'` is equivalent to `'a', 'b', 'c', 'd'`
 * `'z' downTo 's' step 2` is equivalent to `'z', 'x', 'v', 't'`
 
+## Conditional expressions and ranges practice
+
+### Exercise 1 {initial-collapse-state="collapsed" id="conditional-expressions-exercise-1"}
+
+Create a simple game where you win if throwing two dice results in the same number. Use `if` to print `You win :)`
+if the dice match or `You lose :(` otherwise.
+
+<deflist collapsible="true">
+    <def title="Hint">
+        Use the <a href="operator-overloading.md#equality-and-inequality-operators"> equality operator </a>  (<code>==</code>) to compare the dice results. 
+    </def>
+</deflist>
+
+|---|---|
+```kotlin
+fun main() {
+    val firstResult = Random.nextInt(6)
+    val secondResult = Random.nextInt(6)
+    // Write your code here
+}
+```
+{validate="false" kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-tour-control-flow-conditional-exercise-1"}
+
+|---|---|
+```kotlin
+fun main() {
+    val firstResult = Random.nextInt(6)
+    val secondResult = Random.nextInt(6)
+    if (firstResult == secondResult)
+        println("You win :)")
+    else
+        println("You lose :(")
+}
+```
+{initial-collapse-state="collapsed" collapsed-title="Example solution" id="kotlin-tour-control-flow-conditional-solution-1"}
+
+### Exercise 2 {initial-collapse-state="collapsed" id="conditional-expressions-exercise-2"}
+
+Using a `when` expression, update the following program so that when you input the names of GameBoy buttons, the actions
```

Comment:
You mean "X = jump", "Y = shoot fire", as in Mario Bros, right? :P

Reply:
No. This is for pokemon! :D